lucet-runtime/lucet-runtime-internals/src/instance/signals.rs

```diff
@@ -255,11 +254,7 @@ extern "C" fn handle_signal(signum: c_int, siginfo_ptr: *mut siginfo_t, ucontext
     });
 
     if switch_to_host {
-        HOST_CTX.with(|host_ctx| unsafe {
-            Context::set_from_signal(&*host_ctx.get())
-                .expect("can successfully switch back to the host context");
-        });
-        unreachable!()
+        ctx.set_ip(Context::set_from_signal as *const c_void);
```

Comment:
This comes from `lucet-runtime/lucet-runtime-internals/src/sysdeps/linux.rs` I believe? As a reader, it might be nice to have a comment explaining why this is? (i.e., "_we can use this on other platforms_", or "_this function is linux specific to begin with_"?)

Reply:
We talked about this one directly, this was a misunderstanding on my part. Resolved :+1: